salt/modules/win_wusa.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Update files management via wusa.exe
+
+:maintainer:    Thomas Lemarchand
+:platform:      Windows
+:depends:       PowerShell
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+import salt.utils.platform
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'win_wusa'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if not salt.utils.platform.is_windows():
+        return False, 'Only available on Windows systems'
+
+    powershell_info = __salt__['cmd.shell_info']('powershell', True)
```

Comment:
Please use named parameters.  Some times the modules have their param order changed.

Reply:
Done.